piece_of_cake.py

```diff
@@ -0,0 +1,14 @@
+# Arguments:
+# prices - dictionary[key: item's name, value: price for 100g]
+# optionals - list of ingredients to ignore.
+# ingredients - dictionary[ingredient_name : amount in grams]
+# Return value - float
+# This functions calculates the total price for wanted ingredients
+def get_recipe_price(prices, optionals=[], **ingredients):
```

Comment:
Setting an empty list as a default parameter will result bugs - read this and change the code 
https://docs.python-guide.org/writing/gotchas/